package.json

```diff
@@ -1,28 +1,30 @@
 {
-  "name": "octoclairvoyant",
+  "name": "octoclairvoyant-webapp",
   "version": "1.0.0",
   "private": true,
+  "engines": {
+    "npm": "^8.0.0"
```

Comment:
This is just a hint, it won't actually throw an error if the package manager + version is not matched. When we upgrade to Node v16, it might be useful to set up [Corepack](https://nodejs.org/dist/latest-v16.x/docs/api/corepack.html).

Reply:
The hinting might be helping Renovate bot to find the right package manager to run, so let's leave it in.

That corepack thing seems nice:

> 
> This feature simplifies two core workflows:
> 
> - It eases new contributor onboarding, since they won't have to follow system-specific installation processes anymore just to have the package manager you want them to.
> 
> - It allows you to ensure that everyone in your team will use exactly the package manager version you intend them to, without them having to manually synchronize it each time you need to make an update.

It seems `corepack` is still experimental, so we should probably wait until the docs say it's stable. 😄